src/Shared/BuildEnvironmentHelper.cs

```diff
@@ -459,8 +459,8 @@ internal static void ResetInstance_ForUnitTestsOnly(BuildEnvironment buildEnviro
 
         private static class BuildEnvironmentHelperSingleton
         {
-            // Explicit static constructor to tell C# compiler
```

Comment:
What is this? It looks like you deleted a space and replaced it with a space?

Reply:
It was apparently a "NO-BREAK SPACE", which was replaced with a "SPACE".